background/sync-manager.js

```diff
@@ -192,21 +202,6 @@ const syncMan = (() => {
     });
   }
 
-  async function handle401Error(err) {
-    let authError = false;
-    if (err.code === 401) {
-      await tokenMan.revokeToken(currentDrive.name).catch(console.error);
-      authError = true;
-    } else if (/User interaction required|Requires user interaction/i.test(err.message)) {
```

Comment:
Where are you catching this case? `isGrantError` is only checking for a 401 status

Reply:
Why are you commenting in removed code?

Reply:
This case no longer exists. We disallow implicit auth in `token-manager.js` and stop background sync if `!status.login`.